dev/com.ibm.ws.security.acme/resources/com/ibm/ws/security/acme/resources/AcmeMessages.nlsprops

```diff
@@ -131,7 +131,7 @@ CWPKI2023E=CWPKI2023E: The ACME service could not write to the account key file
 CWPKI2023E.explanation=The account keys could not be stored in the account key file. This can occur if the file permissions are incorrect or the file does not exist.
 CWPKI2023E.useraction=Review the error message for details on the failure. Verify the file location is correct and the server has write file permissions.
 
-CWPKI2024E=CWPKI2024E: The ACME service failed to revoke the requested certificate for the ACME certificate authority at the {0} URI. The certificate is serial number {1}. The error is ''{2}''.
+CWPKI2024E=CWPKI2024E: The ACME service failed to revoke the requested certificate for the ACME certificate authority at the {0} URI. The certificate is serial number {1}. The error is ''{2}''. This can occur if the directoryURI has changed, and if so, it can be ignored.
```

Comment:
Should

> This can occur if the directoryURI has changed, and if so, it can be ignored.

be

> This can occur if the directory URI has changed, and if so, it can be ignored.